packages/@pollyjs/adapter-node-http/src/utils/transport-wrapper.js

```diff
@@ -0,0 +1,206 @@
+import http from 'http';
+import zlib from 'zlib';
+
+import { URL } from '@pollyjs/utils';
+
+const NATIVE_REQUEST = Symbol();
+const NATIVE_END = Symbol();
+
+const LISTENERS = Symbol();
+
+export default class TransportWrapper {
+  constructor(transport, { name, adapter }) {
+    this.name = name;
+    this.adapter = adapter;
+    this.transport = transport;
+  }
+
+  isPatched() {
+    return !!this.transport[NATIVE_REQUEST];
+  }
+
+  patch() {
+    // make sure it's not already patched
+    this.adapter.assert(
+      `Transport ${this.name} can not be patched multiple times`,
```

Comment:
`The ${this.name} transport has already been patched, please stop any running Polly instances.`

Reply:
Done in a3f7248831b7540af93d7a4542ec7579a863a5c2